Public/js/reserve-app.js

```diff
@@ -0,0 +1,182 @@
+        var input = 'Open';
+        var closer = '</button></td>'
+        for(var c=0;c<response.length;c++) {
+          if(response[c].machine == b && response[c].hour == a) {
+            opener = '<td style="background-color:orange;font-size:18px;color:white;padding-top:8px;">';
+            input =   response[c].user;
+            closer = '</td>';
+          }
+        }
+        html += opener;
+        html +=   input;
+        html += closer;
+      }
+      html += '</tr>'
+      $('#table'+table).append(html);
+    }
+  };
+
+  var getReservationData = function(date,table) {
+    date = date.replace(' ','-');
+    date = date.replace(' ','-');
+    date = date.replace(' ','-');
+
+    $.ajax({
+      type: 'GET',
+      url: 'reservations?day=' + date,
+      dataType: 'json',
+      success: function(response) {
+        if(table == 1) {
+          fillTable(response,table,parseInt(new Date().toString().slice(16,18)));
+        } else {
+          fillTable(response,table,0);
+        }
+      }
+    });
+  };
+
+  var logOut = function() {
+    $.ajax({
+      type: 'DELETE',
+      url: 'sessions',
+      dataType: 'json',
+      success: function(response) {
+        if(response.ok || response.message) {
+          window.location.href = "/";
+        } else {
+          console.log(response);
+        }
+      }
+    });
+  }
+
+  var moveOn = function(url) {
+    window.location.href = "/"+url;
+  };
+
+  var runError = function(input,section) {
+    $('#error').remove()
+    var html = '';
+    html += '<p id="error">';
+    html +=   input;
+    html += '</p>';
+    $($('section')[section]).prepend(html)
+  }
+
+  Reservation.prototype.getValues = function(button,table) {
+    var hour;
+    var machine;
+    var ind;
+    for(var a=1;a<24;a++) {
+      if($(button).parent().parent()[0] == $(table+' tr')[a]) {
+        ind = a;
+      }
+    }
+    $(table).find('tr').each(function (i, el) {
+      if(i == ind) {
+        hour = $($(this).find('td')[0]).text().slice(0,2);
+        var tds = $(this).find('td').each(function(index,elem){
+          if(elem == $(button).parent()[0]) {
+            machine = index;
+          }
+        });
+      }
+    });
+    this.machine = machine;
+    this.hour = hour;
+  }
+
+  $(document).on('click','.res',function() {
+    var newReservation = new Reservation();
+    if($(this).parent().parent().parent().parent().attr('id') == 'table1') {
+      newReservation.day = today.slice(0,15).replace(' ','-').replace(' ','-').replace(' ','-');
+      newReservation.getValues(this,'#table1');
+    } else {
+      newReservation.day = tmrw.replace(' ','-').replace(' ','-').replace(' ','-');
+      newReservation.getValues(this,'#table2');
+    }
+    newReservation.makeReservation();
+  });
+
+  $(document).on('click','#log-out',function() {
+    logOut();
+  });
+
+  $(document).on('click','a',function() {
+    if(this == $('a')[0]) {
```

Comment:
:8ball: 

Seems like you could pass  `this` to the `moveOn` method and have an specific behaviour on each link?